Mathlib/Topology/ContinuousFunction/Basic.lean

```diff
@@ -351,6 +374,17 @@ theorem pi_eval (f : ∀ i, C(A, X i)) (a : A) : (pi f) a = fun i : I => (f i) a
 def eval (i : I) : C(∀ j, X j, X i) where
   toFun := Function.eval i
 
+variable (A X) in
+/--
+Giving a continuous map out of a disjoint union is the same as giving a continuous map out of
+each term
+-/
```

Comment:
This possibly deserves a reference to `Equiv.curry`; I'll let another reviewer decide.

Reply:
Nevermind, that's not relevant here